github.yaml

```diff
@@ -693,11 +693,9 @@ env:
     required: false
     sensitive: false
     description: GitHub Enterprise Server hostname (e.g. https://ghes.example.com). Optional, for GitHub Enterprise integration.
-args:
-  - run
-  - -i
-  - --rm
-  - -e
-  - GITHUB_PERSONAL_ACCESS_TOKEN
-  - ghcr.io/obot-platform/mcp-images-github:main
-command: docker
+
+runtime: containerized
+containerizedConfig:
+  image: ghcr.io/obot-platform/mcp-images-github:main
+  path: "/sse"
```

Comment:
I know any path works, but I think it is confusing to use `/sse` here because it's not an SSE server.

Reply:
Oh, yeah I only did that because I knew that was the old default path that was hardcoded in obot itself. I did this before I saw your comment that we don't need that. I'll change this to just `/`.